.github/workflows/release_production.yml

```diff
@@ -1,4 +1,8 @@
 name: Deriv App Production Workflow
+
+permissions:
+  contents: write
+
 on:
```

Comment:
Hi @aswathy-deriv let's try with test.yml, if we need write, or read is enough

Reply:
Also, can we put the permisison on job level, like @rupato-deriv's PR?

Reply:
But @habib-deriv ,actually its written that  each workflow file to define the minimum required permissions: block at the top level (before jobs:)
<img width="991" alt="Screenshot 2025-06-18 at 2 58 42 PM" src="https://github.com/user-attachments/assets/477c922c-ede8-40bb-a9fb-6c59a39e21a4" />